setup_chef_cookbooks.sh

```diff
@@ -71,3 +71,4 @@ if [[ ! -d kafka ]]; then
 fi
 [[ -d jmxtrans ]] || git clone https://github.com/jmxtrans/jmxtrans-cookbook.git jmxtrans
 [[ -d cobblerd ]] || git clone https://github.com/cbaenziger/cobbler-cookbook.git cobblerd -b cobbler_profile
+[[ -d chef-vault ]] || git clone https://github.com/opscode-cookbooks/chef-vault.git chef-vault
```

Comment:
Ray, is there any reason the git clone of `chef-vault` is needed rather than a `knife cookbook download` of it? Usually this is only done for cookbooks which are not in the [supermarket](https://supermarket.chef.io/cookbooks/chef-vault); right now both look to be at version 1.3.0.